File: iniciante/python/1117-validacao-de-nota.py
def validacao_de_nota():
    notas_validas = soma = 0
    while True:
        if notas_validas == 2:
            break
        nota = float(input())
        if 0 <= nota <= 10:
            soma += nota
            notas_validas += 1
        else:
            print('nota invalida')
    media = soma / 2
    print(f'media = {media:.2f}')


validacao_de_nota()

File: iniciante/c/1117-validacao-de-nota.c
#include <stdio.h>

int main(){
    int i = 0;
    float nota, soma = 0, media;
    while (i < 2){
        scanf("%f", &nota);
        if (nota < 0.0 || nota > 10.0){
            printf("nota invalida\n");
        }
        else{
            soma += nota;
            i++;
        }
    }
    media = soma / 2;
    printf("media = %.2f\n", media);

    return 0;
}
